int32 pwm_1
int32 pwm_2
int32 pwm_3
int32 pwm_4
